unlock-app/src/middlewares/walletMiddleware.js

```diff
@@ -218,6 +219,21 @@ const walletMiddleware = config => {
               action.price
             )
           })
+        } else if (action.type === SIGN_DATA) {
+          const { data } = action
+          walletService.signDataPersonal(
+            '', // account address -- unused in walletService
```

Comment:
What would this be used for in the other cases? This is not on you but kind of weird